.github/workflows/checks.yml

```diff
@@ -32,6 +32,17 @@ jobs:
           echo "$HOME/.cargo/bin" >> $GITHUB_PATH
           . "$HOME/.cargo/env"
           rustup install
+          curl -fsSL https://github.com/tamasfe/taplo/releases/latest/download/taplo-full-linux-x86_64.gz \
+            | gzip -d - | sudo install -m 755 /dev/stdin /usr/local/bin/taplo
+
+      - name: Check TOML format
+        run: |
+          taplo format --check
+          echo "Please run `taplo format` to fix any TOML formatting issues"
```

Comment:
Does this check work?
imho, exit code equals `0` for this check always.

Reply:
If you modify toml formatting and run this you will get status code 1

Reply:
yes
```bash
$ taplo format --check >/dev/null 2>&1
$ echo $?
1
```

but in this case

```bash
$ taplo format --check >/dev/null 2>&1; echo "Please run taplo format to fix any TOML formatting issues"
Please run taplo format to fix any TOML formatting issues
$ echo $?
0
```

Reply:
Also, the call
``` 
echo "Please run `taplo format` to fix any TOML formatting issues"
```
calls `taplo format` one more time

Reply:
The step fails if `taplo format --check` finds formatting issues because GitHub Actions uses `set -e` by default, which causes the script to exit immediately on any error. The echo line won't run if the previous command fails. Please check https://github.com/QuantumFusion-network/qf-solochain/actions/runs/14337195443/job/40187203534?pr=69.

Reply:
This is a copy from polkadot-sdk workflow.
https://github.com/paritytech/polkadot-sdk/blob/010fd7bd1cd586bbbe4a4c48cd631e2ec263af5e/.github/workflows/checks-quick.yml#L75-L78

Reply:
Thanks for the thorough review by the way!

Reply:
`echo` removed, step name seems sufficient to understand the source of the error in case the reader does not know what is `taplo`.